example/lib/code_snippet_button.dart

```diff
@@ -78,7 +78,8 @@ class _CodeDialog extends StatelessWidget {
               child: Column(
                 mainAxisSize: MainAxisSize.min,
                 children: [
-                  YaruAnimatedNoNetworkIcon(
+                  YaruAnimatedIcon(
+                    const YaruAnimatedOkIcon(),
```

Comment:
I think you should use                   `YaruAnimatedNoNetworkIcon` here 🙂

Reply:
woops